seed/views/00100_indivdual_capacity_view.js

```diff
@@ -0,0 +1,19 @@
+exports.seed = function (knex, Promise) {
+  var sql = `CREATE VIEW app.individual_capacity_view 
+    WITH SCHEMABINDING 
+    AS 
+    SELECT 
+      wr.effective_from
+      , wpc.total_points
+      , wpc.available_points
+      , wpc.reduction_hours
+      , w.workload_owner_id AS id
+    FROM app.workload_points_calculations wpc
+      JOIN app.workload w ON wpc.workload_id = w.id
+      JOIN app.workload_report wr ON wpc.workload_report_id = wr.id;`
```

Comment:
Does this need to filter on the workload report effective_to?  Making sure we use only the current workload report?

Reply:
As far as I know, capacity uses old reports as well. Looking for a date range rather than the current. (Date range applied in query in wmt-web)